CIP-1694/README.md

```diff
@@ -1292,6 +1292,7 @@ The following table details the ratification requirements for each governance ac
 | 5<sub>b</sub>. Protocol parameter changes, economic group         | ✓  | $P_{5b}$ | \-       |
 | 5<sub>c</sub>. Protocol parameter changes, technical group        | ✓  | $P_{5c}$ | \-       |
 | 5<sub>d</sub>. Protocol parameter changes, governance group       | ✓  | $P_{5d}$ | \-       |
+| 5<sub>e</sub>. Protocol parameter changes, security group         | \- | \-       | $Q_{5e}$ |
```

Comment:
```suggestion
| 5<sub>e</sub>. Protocol parameter changes, security group         | ✓ | $P_{5e}$  | $Q_{5e}$ |
```
Does CC and DRep vote not count at all for these parameters?

Reply:
This security group is exclusive to SPOs. DReps and CC votes will count, because the same parameters will belong to different DRep's groups and all parameter changes need full CC agreement.

In other words: 
* For DReps: `allPParams == union [network group, economic group, technical group, governance group]`
* For SPOs: `isSubsetOf (security group) allPParams`

Reply:
Ahh understood, the parameters are in multiple groups, that was not clear to me.
Might want to update the table somehow to reflect this as it is misleading at the moment.

Reply:
You don't think it is enough to actually look at the params in the group, there is list below in this PR:

The **security group** consists of:
* `maxBBSize`
* `maxTxSize`
* `maxBHSize`
* `maxValSize`
* `maxBlockExUnits`
* `minFeeA`
* `minFeeB`
* `coinsPerUTxOByte`
* `govActionDeposit`

Reply:
There was also a paragraph explaining that membership in the security group does not prevent membership in other groups. But since no one read that & everyone got confused instead, it's now called 'security relevant protocol parameters'. There's no difference and the implementation will be the same, but hopefully it'll be less confusing.